### PR TITLE
Breadcrumbs Background Modifiers

### LINKED ISSUE #301: Breadcrumbs is only ever turquoise
### Component in question:
- `Breadcrumbs`

### Description of issue:
- Breadcrumbs should recognize the modifiers for `whats-on`, `locations`, and the other section colors and have its background update accordingly.

### PR DESCRIPTION
**Pull request by helenvholmes**
Issue number: #301 + #283 

## **This PR does the following:**
- Adds modifiers for Breadcrumbs: `--locations`, `--whats-on`, `--books-and-more`, `--research`
- Updates stories for Breadcrumbs: adds designs, adds knobs

### Front End Review:
- [ ] View [the Breadcrumbs in Storybook](https://deploy-preview-312--stoic-murdock-c7f044.netlify.app/?path=/story/breadcrumb--breadcrumbs)
- [ ] View [Colors in Storybook](https://deploy-preview-312--stoic-murdock-c7f044.netlify.app/?path=/story/colors--colors-brand)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 Currently, this repo is in Prerelease. When it is released, this project will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 ========
 
+## 0.9.1
+
+### Changes
+
+-   `Breadcrumbs`'s background color now reflects the app's globally applied section modifier, such as `.nypl--locations`. It recognizes these for `--locations`, `--whats-on`, `--research`, and `--books-and-more`.
```
Comment (hokei) [inline CHANGELOG.md:13]: Does this need to be changed to reflect https://github.com/NYPL/nypl-design-system/issues/301#issuecomment-666517996

Comment (hokei) [inline CHANGELOG.md:13]: Does this need to be changed to reflect https://github.com/NYPL/nypl-design-system/issues/301#issuecomment-666517996

File: src/components/Breadcrumbs/_Breadcrumbs.scss [hunk 2]
```diff
@@ -3,67 +3,83 @@ $break-breadcrumbs-mobile: max-width 599px;
...
-}
 
-.breadcrumbs__link,
-.breadcrumbs__link:link,
-.breadcrumbs__link:visited,
-.breadcrumbs__link:focus {
-    color: inherit;
-    text-decoration: none;
-}
+    .nypl--books-and-more & {
```
Comment (hokei) [inline src/components/Breadcrumbs/_Breadcrumbs.scss:70]: So this is going to be storybook-only?

File: CHANGELOG.md (hunk 1 above)
Comment (helenvholmes) [reply CHANGELOG.md:13]: That is what this comment is saying, but your question illustrates that that's not clear. How would you word this?

File: src/components/Breadcrumbs/_Breadcrumbs.scss (hunk 2 above)
Comment (helenvholmes) [reply src/components/Breadcrumbs/_Breadcrumbs.scss:70]: The styles are getting shipped, so no. In order to get these styles into your app you'll need to apply `.nypl--books-and-more` to `<body>` in your `App.js` or something along those lines. I think an update to the README would be appropriate, potentially.

File: CHANGELOG.md (hunk 1 above)
Comment (hokei) [reply CHANGELOG.md:13]: OHHH ok so what I understand (after looking at your comment below) is that you now will have 
```
<body className="books-and-more">
<DS.breadcrumb /> 
</body> 
```

And these styles will be applied.  

Without the comment below, it seemed like you needed to pass in 
`<DS.Breadcrumb className="books-and-more" />` which is not what we want.  

I think updates to the Readme on what the section globals are and how to use them might be useful here.

**Comment by helenvholmes** [comment]
@hokei instead of adding documentation to the README, I've built it into the `Colors.stories.tsx`—let me know what you think.

**Review by nypl-wluisi — APPROVED** [review]
This looks good to me.

**Pull request merged**